sitejava.js

```diff
@@ -0,0 +1,160 @@
+let fetchPromise = fetch('https://newsapi.org/v2/top-headlines?country=gb&apiKey=56887af3601845008f0dc6fa7f8b8810') //fetchs default page -news in UK
+
+
+const Buttons = document.querySelectorAll(".normButton")
+const categoryButtons = document.querySelectorAll(".categoryButton")
+const paginationButton = document.querySelector(".pagButton")
+const form = document.querySelector("form")
+const textArea = document.querySelector(".textarea")
+
+
+let textInBox = "" // variable to hold the value input to the text area
+textArea.addEventListener("input", function(event) {
+  textInBox = event.target.value
```

Comment:
It might be easier to grab the text input content, once submit is detected. It will save writing an additional event handler and save using a global variable